stock_operating_unit/model/stock.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp.tools.translate import _
+from openerp import api, fields, models
+from openerp.exceptions import Warning as UserError
+
+
+class StockWarehouse(models.Model):
+    _inherit = "stock.warehouse"
+
+    operating_unit_id = fields.Many2one('operating.unit',
+                                        'Operating Unit',
+                                        default=lambda self:
+                                        self.env['res.users'].
+                                        operating_unit_default_get(self._uid))
+
+    @api.multi
+    @api.constrains('operating_unit_id', 'company_id')
+    def _check_company_operating_unit(self):
+        for rec in self:
+            if rec.company_id and rec.operating_unit_id and rec.company_id \
+                    != rec.operating_unit_id.company_id:
+                raise UserError(_('Configuration error!\nThe Company in the\
+                Stock Warehouse and in the Operating Unit must be the same.'))
```

Comment:
This string contains a lot of spaces which is then displayed weirdly in the error message. You should use the string concatenation:

```python

raise UserError(
    _('Configuration error\nThe Company in the '
      'Stock Warehouse and in the Operating Unit must be the same.')
)

```

(note: I also removed the exclamation point in the message which is not recommended (https://cybertext.wordpress.com/2011/11/18/error-messages-that-make-you-feel-stupid/)